build.gradle

```diff
@@ -39,6 +41,17 @@ repositories {
   maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
 }
 
+allprojects {
+  group 'org.opensearch.index.codec.customcodecs'
```

Comment:
Shouldn't the group be org.opensearch.plugin (https://github.com/opensearch-project/OpenSearch/blob/main/plugins/build.gradle)?

Reply:
I'm honestly not sure. I followed the convention here: https://github.com/opensearch-project/common-utils/blob/main/build.gradle#L44

Reply:
I think it should be `org.opensearch` per the example here: https://github.com/opensearch-project/opensearch-plugins/blob/main/BUILDING.md#define-a-version-based-on-the-opensearch-dependency

Reply:
Right https://github.com/opensearch-project/index-management/blob/main/build.gradle#L192